client/blocks/plan-thank-you-card/index.jsx

```diff
@@ -44,6 +45,13 @@ const PlanThankYouCard = ( {
 
 		return null;
 	};
+	const renderDescription = () => {
+		if ( description ) {
```

Comment:
It comes in as a prop. I've updated the PlanThankYouCard after some feedback in Slack from you.